engine/runtime/src/test/java/org/enso/interpreter/test/DebuggingEnsoTest.java

```diff
@@ -113,6 +113,8 @@ private Value createEnsoMethod(String source, String methodName) {
   @Test
   public void recursiveFactorialCall() {
     final Value facFn = createEnsoMethod("""
+    from Standard.Base.Data.Ordering import all
+    
```

Comment:
Without this import we cannot use `<=` anymore.
- result of 0df66b108060
- also reported as #5610